### DISCUSSION TITLE
Hyprbars icons not centered in the buttons

### DISCUSSION THREAD

**Discussion by winkelnp** [hyprwm/Hyprland#8796] [Q&A]
I have hyprbars on a toggle for the case when my laptop is a little further away and I can't access the keyboard, and it works amazing, just that the icons are offset to the bottom left, which is visually a little off-putting. Is there a way for me to fix this? I must be configuring something wrong I don't see this happening on other implementations…

![image](https://github.com/user-attachments/assets/986f34f7-3bfd-4ee9-bdff-aed0f1896fc7)

My config:
```
    hyprbars {
        bar_color = rgb(242424)
        bar_height = 30
        # col.text =
        bar_title_enabled = true
        # bar_text_size = 10
        bar_text_font = Fira Mono
        bar_text_align = center # [center/left]

        bar_buttons_alignment = right # [right/left]
        bar_part_of_window = true
        bar_precedence_over_border = true
        bar_padding = 7
        bar_button_padding = 10
        # hyprbars-button = color, size, icon, on-click
        hyprbars-button = rgb(fe5e56), 15, , hyprctl dispatch killactive
        hyprbars-button = rgb(fdba2d), 15, , hyprctl dispatch fullscreen 1
        hyprbars-button = rgb(28c640), 15, , hyprctl dispatch movetoworkspacesilent special:magic
    }
```

**Comment by vaxerski** [comment]
` hyprbars-button = rgb(fe5e56), 15, , hyprctl dispatch killactive`

have you tried making the size even?

**Reply by winkelnp** [reply]
Confusingly this appears to have semi-fixed itself (hadn't checked back yet, the shift on size 15 is less egregious, but it is better on size 16. Thanks for the tip!
